setup.py

```diff
@@ -19,6 +19,7 @@
 
 import os
 import setuptools
+import time
```

Comment:
I'm fine to make the change, but why would time be imported before setuptools?
What's the principle?

Per the [style guide](https://google.github.io/styleguide/pyguide.html#313-imports-formatting):

> Within each grouping, imports should be sorted lexicographically, ignoring case

Reply:
On my machine pylint was complaining about the ordering, I presume because time is python standard library import (3.13.2 in style guide), while I guess setuptools isn't considered one. This happens when I would directly run

`pylint setup.py`

Our travis run doesn't catch this actually, since we [run pylint over gcp_variant_transforms](https://github.com/googlegenomics/gcp-variant-transforms/blob/master/run_presubmit.sh#L32) subdirectory, and setup.py lies outside of it. It's fine though, I can fix it later. Thanks for your contribution, it helped us a lot of with fixing weird python 3 errors!